Archloom/Multiplatform Port

UnlikePaladin opened the pull request:
Supports and builds for Forge and Fabric
Pls review

UnlikePaladin committed:
WIP Port

UnlikePaladin committed:
figura now fully multiloader

UnlikePaladin committed:
Merge remote-tracking branch 'upstream/1.20' into 1.20

# Conflicts:
#	common/src/main/resources/assets/figura/textures/gui/toast/fran.png

UnlikePaladin committed:
Appended Mod loader at the end of jar file
Fixed Action Wheel and Paperdoll not rendering on Forge
Fixed a Crash when using Commands on Forge
Added Actions building on Push

skyrina commented:
read through the code and everything looks good, can someone verify that their avatar works correctly on both fabric and forge?
you can find download links to the jars [here](https://github.com/FiguraMC/Figura/actions/runs/5566040000)

Covkie commented:
java/org/moon/figura should be changed to java/org/figuramc/figura

UnlikePaladin commented:
I'll change it quickly before merging

UnlikePaladin committed:
Repackaged to FiguraMC
Changed fabric.mod.json and mods.toml to point to new repo
Updated the backend ip
